src/main/java/reposense/commits/CommitInfoAnalyzer.java

```diff
@@ -68,7 +68,8 @@ public static CommitResult analyzeCommit(CommitInfo commitInfo, Map<String, Auth
             logger.log(Level.WARNING, "Unable to parse the date from git log result for commit.", pe);
         }
 
-        String message = elements[MESSAGE_INDEX];
+        boolean containsMessage = elements.length > MESSAGE_INDEX;
+        String message = (containsMessage) ? elements[MESSAGE_INDEX] : "";
```

Comment:
I think we can put this 2 into a single line

Reply:
Do you mean 'String message = (elements.length > MESSAGE_INDEX) ? elements[MESSAGE_INDEX] : "";'?

Reply:
Yes